front/src/pages/Home/Home.tsx

```diff
@@ -26,23 +31,65 @@ const Home = (): JSX.Element => {
           }));
         }
       },
+      refetchOnWindowFocus: false,
     }
   );
 
-  if (isFetching) {
-    return <Loader />
+  const updateTodoMutation = useMutation(
```

Comment:
looks good. optimistic updates rock!
I propose to extract this logic into custom hooks `useFetchTodos` and `useUpdateTodo` so hide implementation details from component.

Reply:
@skyboyer 
Done.